README.md

```diff
@@ -122,6 +124,18 @@ Found 2 dependencies with mismatching versions across the workspace. Fix with `-
 | `--ignore-path` | Workspace-relative path of packages to ignore mismatches for (option can be repeated). |
 | `--ignore-path-pattern` | RegExp of workspace-relative path of packages to ignore mismatches for (option can be repeated). |
 
+## Node API
+
+| Function | Description |
+| :-- | :-- |
+| `check(path, options)` | Checks for inconsistencies across a workspace. Optionally fixes them. Returns lists of inconsistencies: a complete list, a fixable list, and an un-fixable list. |
+| `mismatchingVersionsToDetailedSummary(versions)` | Returns a string of human-readable tables describing mismatching dependency versions. |
+| `mismatchingVersionsToFixedSummary(versions)` | Returns a string summary of the mismatching dependency versions that were fixed. |
+
+More information about parameters and return values can be found in the types and JSDoc comments.
```

Comment:
The documentation here seems insufficient for an API. The user shouldn't have to look through types and comments to determine parameters.

Reply:
Added more detail to README.